app/src/routes/auth/authRoutes.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
+
+import express from "express";
+import ctrl from "./authCtrl.js"
+import jwt from "jsonwebtoken";
+
+const router = express.Router();
+
+router.post("/login", (req, res, next) => {
+  let token = req.headers['authorization'];
```

Comment:
```suggestion
  const token = req.headers['authorization'];
```
let으로 선언할 이유가 없어 보여요